packages/@webex/internal-plugin-metrics/src/index.ts

```diff
@@ -7,11 +7,16 @@ import '@webex/internal-plugin-device';
 import {registerInternalPlugin} from '@webex/webex-core';
 
 import Metrics from './metrics';
+import NewMetrics from './new-metrics';
+import {userAgentToString} from './call-diagnostic/call-diagnostic-metrics.util';
 import config from './config';
+import {ClientEvent} from './call-diagnostic/types/ClientEvent';
 
 registerInternalPlugin('metrics', Metrics, {
   config,
 });
 
+// eslint-disable-next-line no-restricted-exports
 export {default, getOSNameInternal} from './metrics';
-export {config};
+export {config, NewMetrics, userAgentToString};
```

Comment:
just to confirm, once you do all these changes, we will be exporting only the NewMetrics and not the old ones, right?

Reply:
yes